source-hackernews/source_hackernews/__init__.py

```diff
@@ -0,0 +1,67 @@
+from logging import Logger
+from typing import Callable, Awaitable
+
+from estuary_cdk.flow import (
+    ConnectorSpec,
+)
+from estuary_cdk.capture import (
+    BaseCaptureConnector,
+    Request,
+    Task,
+    common,
+    request,
+    response,
+)
+from estuary_cdk.http import HTTPMixin
+
+from .resources import all_resources
+from .models import (
+    ConnectorState,
+    EndpointConfig, ResourceConfig
+)
+
+
+class Connector(
+    BaseCaptureConnector[EndpointConfig, ResourceConfig, ConnectorState],
+    HTTPMixin,
+):
+    def request_class(self):
+        return Request[EndpointConfig, ResourceConfig, ConnectorState]
+
+    async def spec(self, log: Logger, _: request.Spec) -> ConnectorSpec:
+        return ConnectorSpec(
+            configSchema=EndpointConfig.model_json_schema(),
+            documentationUrl="https://docs.estuary.dev",
```

Comment:
FYI if you want to have actual docs for this connector, this is the URL to update for that.